pkgs/by-name/sx/sx/package.nix

```diff
@@ -27,14 +30,33 @@ stdenvNoCC.mkDerivation rec {
 
   postInstall = ''
     patsh -f $out/bin/sx -s ${builtins.storeDir}
+
+    install -Dm755 -t $out/share/xsessions ${
```

Comment:
`copyDesktopItems` does not work here?

Reply:
No because we need to copy to a custom location (`$out/share/xsession`).

Reply:
I would suggest a let-in for better readability, but it's nothing serious.